restapi/auth.go

```diff
@@ -53,16 +52,10 @@ func (api api) Login(w http.ResponseWriter, r *http.Request) render.Renderer {
 }
 
 func generateJWTToken(email string, w http.ResponseWriter) error {
-	jwtExpiry, err := config.GetTokenExpiryDuration()
-	if err != nil {
-		return err
-	}
-	expiration := time.Now().Add(jwtExpiry)
-
 	claim := &models.Claims{
 		Email: email,
 		StandardClaims: jwt.StandardClaims{
-			ExpiresAt: expiration.Unix(),
+			ExpiresAt: 0,
```

Comment:
I set ExpiresAt to 0 so that this JWT token never expires. This is generally bad practice but the reason is that google will do auto refresh it's own token so it may not call our backend endpoint which means that we have an outdated jwt token. To solve this we would need to perform auto refresh on our token as well but that is out of scope right now so doing it the hacky way